week9/answers.sql

```diff
@@ -1,3 +1,167 @@
--- Create your tables, views, functions and procedures here!
+DROP SCHEMA IF EXISTS social;
 CREATE SCHEMA social;
 USE social;
+
+CREATE TABLE users (
+	user_id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT,
+   first_name VARCHAR(255),
+   last_name VARCHAR(255),
+   email VARCHAR(255),
+   created_on DATETIME DEFAULT CURRENT_TIMESTAMP
```

Comment:
You'll want to use a TIMESTAMP rather than a DATETIME (see here: https://wwcc.instructure.com/courses/2267209/pages/dates-and-time?module_item_id=69087028 ) (specifically, the discussion about timezones).

Reply:
This comment is going to be true throughout this project. Be aware that in all cases below, a TIMESTAMP is preferred over a DATETIME to prevent errant behavior by servers when dealing with DST and timezones.